app.json

```
@@ -11,7 +11,7 @@
     ],
     "name": "Wavv",
     "slug": "Wavv",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

```suggestion
    "version": "1.1.0",
```

Since we added few features

---

We released version `1.0.1` on the Apple Store without updating GitHub, which led to this mismatch. We'll use version `1.1.0` for future releases to ensure proper versioning and consistency.